src/System.Private.CoreLib/shared/System/Resources/ResourceManager.NonUap.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+
+namespace System.Resources
+{
+    public partial class ResourceManager
+    {
+        private bool UseUapResourceManagement { get => false; }
+
+        private string GetStringFromPRI(string stringName, CultureInfo culture, string neutralResourcesCulture) { return null; }
+
+        private void SetAppXConfiguration() { }
+    }
```

Comment:
I can get rid of this file and put two `#if`s in ResourceManager.cs instead if that is preferred.